src/import_edf.jl

```diff
@@ -220,23 +216,29 @@ function onda_samples_from_edf_signals(target::Onda.Signal, edf_signals,
         end
         copyto!(view(sample_data, i, :), encoded_samples)
     end
-    return Samples(target, true, sample_data)
+    return Samples(sample_data, target, true)
 end
 
 """
-    import_edf!(dataset::Dataset, edf::EDF.File, uuid::UUID=uuid4();
-                custom_extractors=(), import_annotations::Bool=true)
+    store_edf_as_onda(path, edf::EDF.File, uuid::UUID=uuid4();
+                      custom_extractors=(), import_annotations::Bool=true)
 
-Create, store, and return a `uuid::UUID => recording::Onda.Recording` where all
-the signals in `recording` are converted from the provided `edf`. Collections
-of `EDF.Signal`s are mapped as channels to `Onda.Signal`s via simple "extractor"
-callbacks of the form:
+Convert an EDF.File to `Onda.Samples` and `Onda.Annotation`s, store the samples
+in `\$path/samples/`, and write the Onda signals and annotations tables to
+`\$path/onda.(signals|annotations).arrow`.  Returns `uuid => (signals,
```

Comment:
so `onda.signals.arrow` is technically not a compliant name here - if the caller can't pass in the desired name, should change to `edf.onda.signals.arrow` or `exported.onda.signals.arrow`? (same for `onda.annotations.arrow`)